app/services/message_employee_matcher.rb

```diff
@@ -24,7 +24,10 @@ def match_time(time_zone)
     employee_current_hour = Time.current.in_time_zone(time_zone).hour
     employee_current_minute = Time.current.in_time_zone(time_zone).min
 
-    employee_current_hour == message.time_of_day.hour && employee_current_minute == message.time_of_day.min
+    # Since time is always ticking, we need to check to see if the employee's
```

Comment:
rather than a comment, can we create a test to reproduce this behavior so we will have a test failure if this behavior ever gets reintroduced?